src/loaddb/load_error_handler.cpp

```diff
@@ -92,6 +94,9 @@ namespace cubload
 	err_msg.append (format (get_message_from_catalog (LOADDB_MSG_LINE), get_lineno ()));
 	err_msg.append (std::string (er_msg ()));
 	err_msg.append ("\n");
+
+	// once error message was fetched from error manager, we can clear the error
+	er_clear ();
```

Comment:
please move er_clear to `on_syntax_failure ()`. it doesn't look safe to clear errors in all cases, we might miss something.